pkg/rule/api/v1_test.go

```diff
@@ -178,6 +178,15 @@ func testEndpoints(t *testing.T, api *API) {
 				},
 			},
 		},
+		{
+			endpoint: api.alerts,
+			response: &AlertDiscovery{Alerts: []*Alert{}},
+		},
+		{
+			endpoint: api.stateAlerts,
+			params: map[string]string{"state": "firing"},
+			response: &AlertDiscovery{Alerts: []*Alert{}},
```

Comment:
Could you add proper tests  mock some firing alerts and check that they are returned currectly